build.sbt

```
@@ -81,7 +81,7 @@ lazy val parser = project
     mimaSettings,
     name := "twirl-parser",
     libraryDependencies += "org.scala-lang.modules" %% "scala-parser-combinators" % ScalaParserCombinatorsVersion % Optional,
-    libraryDependencies += "com.novocode"    % "junit-interface" % "0.11"           % Test,
+    libraryDependencies += "com.github.sbt"    % "junit-interface" % "0.13.2"           % Test,
```

```suggestion
    libraryDependencies += "com.github.sbt"  % "junit-interface" % "0.13.2"         % Test,
```

https://app.travis-ci.com/github/playframework/twirl/jobs/545691114#L377

```
[error] 1 files must be formatted
[error] (Compile / scalafmtSbtCheck) 1 files must be formatted
[error] Total time: 0 s, completed Oct 28, 2021, 8:52:15 PM
The command "scripts/validate-code.sh" exited with 1.
```